modules/mixed-arch-modify-machine-set.adoc

```diff
@@ -0,0 +1,76 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-modify-machine-set_{context}"]
+
+= Adding a machine set to your cluster using the `arm64` bootimage  
+
+To add `arm64` worker nodes to your heterogeneous cluster, you must create an Azure machine set that uses the `arm64` bootimage. To create your own custom machine set on Azure, see "Creating a machine set on Azure" in _Additional Resources_. 
+
+.Prerequisites 
+
+* Installed the OpenShift CLI (`oc`). 
+
+.Procedure
+. View the machine sets in the cluster:
++
+[source,terminal]
+----
+oc get machineset -o yaml my-machine-set.yaml
```

Comment:
based on the discussion on slack, this will give a list of machineSets, not only one. We could get rid of it and just continue with the example below.

Reply:
shouldn't this be `oc get machines` ?

Reply:
I got rid of this command and just had the first command be the "Create a machine set" step.